lib/pure/encodings.nim

```diff
@@ -9,13 +9,43 @@
 
 ## Converts between different character encodings. On UNIX, this uses
 ## the `iconv`:idx: library, on Windows the Windows API.
+##
+## The following example shows how to change character encodings. The first
```

Comment:
instead of a large paragraph, split it into short concise comments on the corresponding relevant lines in the runnableExamples (focusing on the non-obvious parts, since the example is already quite self documenting)

one that that's less obvious (and should be explain in comments) is the ordering eg:
```nim
# convert `orig from "UTF-8" to "CP1252"
cp1252 = convert(orig, "CP1252", "UTF-8")
```

Reply:
make sense